lib/src/layer/circle_layer/painter.dart

```diff
@@ -92,6 +93,7 @@ class CirclePainter extends CustomPainter {
       for (final radius in pointsByRadius.keys) {
         final pointsByRadiusColor = pointsByRadius[radius]!;
         final radiusPaint = paint..strokeWidth = radius * 2;
+        radiusPaint.blendMode = BlendMode.src;
```

Comment:
You should move it to first assignment (paintPoint) to avoid useless re-assignment.